agent_build_refactored/managed_packages/scalyr_agent_python3/internal/agent-python3-config.py

```diff
@@ -175,19 +174,7 @@ def get_current_openssl_version(version_type: str) -> Tuple[bool, str]:
     if result.returncode != 0:
         return False, "Not found"
 
-    version_output = result.stdout.decode()
-
-    if version_type == OPENSSL_1_1_1_VERSION:
```

Comment:
I removed this check, so it's now completely up to Python to decide whether the available OpenSSL is valid or not.
Trying to "guess" the output of the openssl version may be a bad idea, for example on  other "unsupported" distributions, which may have valid OpenSSL, but, for some reason have different version format. In this case this check will be just an obstacle which won't let to install the agent on an absolute valid distro.